karma.mc.config.js

```diff
@@ -120,19 +118,20 @@ module.exports = function(config) {
             test: /\.jsx$/,
             exclude: /node_modules/,
             loader: "babel-loader",
-            options: {
-              presets: ["react"], // require("babel-preset-react")
-              plugins: [["transform-object-rest-spread", {"useBuiltIns": true}]]
-            }
```

Comment:
Is that supposed to be part of _this_ PR?

Reply:
sorry, bad rebase – should be fixed now